src/utils.sh

```diff
@@ -454,16 +454,39 @@ function handlers_init() {
 
 export -f handlers_init
 
+function generate_uuid() {
+    source /etc/os-release
```

Comment:
since we need this for other functions, please move it outside of the function scope in case it causes side effects...